prow/github/fakegithub/fakegithub.go

```diff
@@ -507,6 +509,16 @@ func (f *FakeClient) CreateProjectCard(columnID int, projectCard github.ProjectC
 	}
 
 	for project, columnIDMap := range f.ColumnIDMap {
+		for id := range columnIDMap {
+			// Make sure that we behave same as github API
+			// Create project will generate an error when the card already exist in the project
+			if _, exists := columnIDMap[columnID]; exists {
```

Comment:
nit: It is strange to put this conditional inside the loop given that it will evaluate the same for every loop iteration. Please move outside the loop.

Reply:
fixedtrue, fixed.